crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -899,6 +899,9 @@ impl Encryption {
     /// The `secret_storage_key` can be a passphrase or a Base58 encoded secret
     /// storage key.
     ///
+    /// *Note*: This method will create a mark the given `secret_storage_key` as
+    /// the default one if no other default secret storage key exists.
+    ///
```

Comment:
I guess one word is missing from this sentence around “create a mark the given”?

Reply:
Actually `a` should be replaced with `and`. It creates and marks the key as the default one.

Reply:
Fixed by: https://github.com/matrix-org/matrix-rust-sdk/pull/2621/commits/cd84f7412f352b3c45efdc609c0303cd0be2450b.